builder/lxc/config.go

```diff
@@ -29,6 +30,18 @@ type Config struct {
 	ctx interpolate.Context
 }
 
+type ExportConfig struct {
+	Filename string  `mapstructure:"filename"`
```

Comment:
@mhan I believe we need to set struct var first letter in uppercase in order to access the variable. Maybe i'm getting it wrong; 
https://golang.org/ref/spec#Exported_identifiers